test/fixtures.yml

```diff
@@ -760,6 +760,11 @@ paypal_signature:
   password: HBC6A84QLRWC923A
   signature: AFcWxV21C7fd0v3bYYYRCpSSRl31AC-11AKBL8FFO9tjImL311y8a0hx
 
+paypal_commerce_platform:
```

Comment:
why blank?

Reply:
Because this repo is public and in PayPal, in order to have access to PayPal Commerce Platform you have to ask them for it, so I didn't want to someone could have access to their system with our credentials